src/components/Filter/FilterGroup.scss

```diff
@@ -8,3 +8,16 @@
     border-bottom: 1px solid $color-gray-light;
   }
 }
+
+.name {
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.glyph {
+  &:hover {
+    cursor: pointer;
+  }
+  font-size: 0.8em;
+}
```

Comment:
Forslag: Legg til `padding-right: 0.5em` (eller noe annet lurt) for å få litt avstand mellom chevron og groupName.